### PR TITLE
Working pagination

### PR DESCRIPTION
**Pull request by przemyslaw-pacek**
Please check and let me know.

### CODE REVIEW

File: src/common/Pagination/index.js
Comment (bartosz-kasprzyk) [inline src/common/Pagination/index.js:1]: Pagination works great, but I found one issue.
When you're for example on page 8 and you click on a movie/person to see the details and then you click the left arrow on your browser to go back to the list, the url says
`?page=8`
as it should, but the pagination goes back to
`Page 1 of 500`

**Review by bartosz-kasprzyk — CHANGES_REQUESTED** [review]

**Review by VITALIKXXX — APPROVED** [review]

File: src/common/Pagination/index.js
Comment (przemyslaw-pacek) [reply src/common/Pagination/index.js:1]: Already done.

**Review by bartosz-kasprzyk — APPROVED** [review]
Great, now the pagination works perfectly!
The only thing I would've added is a loading state between different pages. It would be great to see the Loading component for half a second and then the next results, but that's a small thing you can change if you have the time to do so. Anyway - good job!

**Comment by bartosz-kasprzyk** [comment]
I've approved the changes, but didn't check the console. Are you getting the same error as I do, or is it only on my end? It looks like this:

error

**Comment by przemyslaw-pacek** [comment]
This is also the case for me, but it is the result of checking pages above 500 that do not return results.

**Pull request merged**